### PR TITLE
Updates to Doc Sections 9 to 12

### PR DESCRIPTION
**Pull request by dakanji**

### CODE REVIEW

File: Docs/Configuration.tex [hunk 1]
```diff
@@ -4117,23 +4117,23 @@ \subsection{Introduction}\label{nvramintro}
...
+Please ensure that variables set in this section do not conflict with items
+in those subsections as the implementation behaviour is undefined otherwise.
 
-For proper macOS functioning it is often required to use \texttt{OC\_FIRMWARE\_RUNTIME}
-protocol implementation currently offered as a part of \texttt{OpenRuntime} driver.
-While it brings any benefits, there are certain limitations which arise depending on the
-use.
+For proper macOS functioning, using the \texttt{OC\_FIRMWARE\_RUNTIME} protocol
+implementation currently offered as a part of the \texttt{OpenRuntime} driver is
+often required. While this typically brings any benefits, there are limitations
```
Comment (PMheart) [inline Docs/Configuration.tex:4129]: Drop `any` maybe?

File: Docs/Configuration.tex [hunk 2]
```diff
@@ -4147,15 +4147,13 @@ \subsection{Properties}\label{nvramprops}
   in \texttt{plist\ multidata} format. GUIDs must be provided in canonic string
   format in upper or lower case (e.g. \texttt{8BE4DF61-93CA-11D2-AA0D-00E098032B8C}).
 
-  Created variables get \texttt{EFI\_VARIABLE\_BOOTSERVICE\_ACCESS} and
-  \texttt{EFI\_VARIABLE\_RUNTIME\_ACCESS} attributes set.
-  Variables will only be set if not present or deleted. I.e. to overwrite
-  an existing variable value add the variable name to the \texttt{Delete} section.
-  This approach enables to provide default values till the operating system
-  takes the lead.
+  The \texttt{EFI\_VARIABLE\_BOOTSERVICE\_ACCESS} and \texttt{EFI\_VARIABLE\_RUNTIME\_ACCESS}
```
Comment (PMheart) [inline Docs/Configuration.tex:4150]: Mark here.

File: Docs/Configuration.tex (hunk 1 above)
Comment (dakanji) [reply Docs/Configuration.tex:4129]: Typo. Supposed to be "many"

File: Docs/Configuration.tex [hunk 3]
```diff
@@ -4694,28 +4693,29 @@ \subsection{Properties}\label{platforminfoprops}
...
-  format is used to store the \texttt{SystemUUID} data. Otherwise, \texttt{Little Endian} is used.
+  the modified DMI tables. If \texttt{UseRawUuidEncoding} is enabled, the \texttt{Big Endian}
+  format is used to store the \texttt{SystemUUID} data. Otherwise, the \texttt{Little Endian}
+  format is used.
 
-  \emph{Note}: Since UUIDs used in DataHub and NVRAM are not standardised
-  and are added by Apple, this preference does not affect them. Unlike
-  SMBIOS they are always stored in the \texttt{Big Endian} format.
+  \emph{Note}: This preference does not affect UUIDs used in DataHub and NVRAM
+  as they not standardised and are added by Apple. Unlike SMBIOS, they are
```
Comment (vit9696) [inline Docs/Configuration.tex:4717]: are not?

File: Docs/Configuration.tex [hunk 4]
```diff
@@ -4777,10 +4777,10 @@ \subsection{Generic Properties}\label{platforminfogeneric}
 
   \begin{itemize}
     \item \texttt{FW\_FEATURE\_SUPPORTS\_CSM\_LEGACY\_MODE} (\texttt{0x1})
-    - Without this bit it is not possible to reboot to Windows installed on
+    - Without this bit, it is not possible to reboot to Windows installed on
       a drive with EFI partition being not the first partition on the disk.
```
Comment (vit9696) [inline Docs/Configuration.tex:4781]: Should add that recent Windows 10 installations are not affected. @khronokernel could you tell us when was that fixed?

File: Docs/Configuration.tex [hunk 5]
```diff
@@ -5996,20 +5994,20 @@ \subsection{APFS Properties}\label{uefiapfsprops}
...
   \texttt{MinDate}\\
   \textbf{Type}: \texttt{plist\ integer}\\
   \textbf{Failsafe}: \texttt{0}\\
   \textbf{Description}: Minimal allowed APFS driver date.
 
-  APFS driver date connects APFS driver with the calendar
-  release date. Older versions of APFS drivers may contain unpatched
-  vulnerabilities, which can be used to inflict harm to the computer.
-  This option permits restricting APFS drivers to only recent releases.
+  The APFS driver date connects the APFS driver with the calendar elease date.
```
Comment (vit9696) [inline Docs/Configuration.tex:6006]: release?

File: Docs/Configuration.tex [hunk 6]
```diff
@@ -5996,20 +5994,20 @@ \subsection{APFS Properties}\label{uefiapfsprops}
...
+  Apple ultimately drops support for APFS drivers from older macOS releases and
```
Comment (vit9696) [inline Docs/Configuration.tex:6007]: It does not. Rather it drops support of old macOS versions. As a result apfs drivers stop getting updated.

File: Docs/Configuration.tex [hunk 7]
```diff
@@ -6028,11 +6026,11 @@ \subsection{APFS Properties}\label{uefiapfsprops}
   \textbf{Failsafe}: \texttt{0}\\
   \textbf{Description}: Minimal allowed APFS driver version.
 
-  APFS driver version connects APFS driver with the macOS
-  release. APFS drivers from older macOS releases will become
-  unsupported and thus may contain unpatched vulnerabilities, which
-  can be used to inflict harm to the computer. This option permits
-  restricting APFS drivers to only modern macOS versions.
+  The APFS driver version connects the APFS driver with the macOS release.
+  Apple ultimately drops support for APFS drivers from older macOS releases and
```
Comment (vit9696) [inline Docs/Configuration.tex:6030]: same as above

File: Docs/Configuration.tex [hunk 8]
```diff
@@ -6117,19 +6115,19 @@ \subsection{Audio Properties}\label{uefiaudioprops}
   \textbf{Failsafe}: \texttt{0}\\
   \textbf{Description}: Minimal heard volume level from \texttt{0} to \texttt{100}.
 
-  Screen reader will use this volume level, when the calculated volume level is
-  less than \texttt{MinimumVolume}. Boot chime sound will not play if the calculated
-  volume level is less than \texttt{MinimumVolume}.
+  Screen reader will use this volume level when the calculated volume level is lower
+  than \texttt{MinimumVolume}. The boot chime will not sound if the calculated
```
Comment (vit9696) [inline Docs/Configuration.tex:6119]: will not be played? I am a bit surprised sound verb is used to describe audio playback.

File: Docs/Configuration.tex [hunk 9]
```diff
@@ -6525,18 +6523,18 @@ \subsection{ProtocolOverrides Properties}\label{uefiprotoprops}
   \textbf{Description}: Replaces Apple audio protocols with builtin
   versions.
 
-  Apple audio protocols allow macOS bootloader and OpenCore to play
+  Apple audio protocols allow OpenCore and the macOS bootloader to play
   sounds and signals for screen reading or audible error reporting.
-  Supported protocols are beep generation and VoiceOver. VoiceOver protocol
+  Supported protocols are beep generation and VoiceOver. Rhw VoiceOver protocol
```
Comment (vit9696) [inline Docs/Configuration.tex:6528]: The?

File: Docs/Configuration.tex (hunk 8 above)
Comment (dakanji) [reply Docs/Configuration.tex:6119]: A chime sounds ... like a bell rings.

Comment (vit9696) [reply Docs/Configuration.tex:6119]: Well, ok, but it is play everywhere below.

Comment (dakanji) [reply Docs/Configuration.tex]: Changed to be same as others

File: Docs/Configuration.tex (hunk 4 above)
Comment (dakanji) [reply Docs/Configuration.tex]: Left alone pending indication from @khronokernel

Comment (vit9696) [reply Docs/Configuration.tex]: We will address it later.

**Pull request merged**